docs/rest/rest-catalog-open-api.yaml

```diff
@@ -94,21 +94,7 @@ paths:
         "
       responses:
         200:
-          description: Server specified configuration values.
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/CatalogConfiguration'
-              example: {
-                "data": {
-                  "overrides": {
-                    "warehouse": "s3://bucket/warehouse/"
-                  },
-                  "defaults": {
-                    "clients": "4"
-                  }
-                }
-              }
+          $ref: '#/components/responses/RESTCatalogConfigResponse'
```

Comment:
I think it is clear that all the request/response objects are for REST, so we can probably omit that from the example and object names.